apps/dashr-svm/README.md

```diff
@@ -0,0 +1 @@
+# dashr-svm
```

Comment:
Let's go ahead and fill this `README.md` with some content so we can 🚀 already 🙂